### PR TITLE
RND-11553 - Adding PyPI Info

### PR DESCRIPTION
**Pull request by EliAsor**

### CODE REVIEW

File: examples/responses/schema.md [hunk 1]
```diff
@@ -38,8 +38,8 @@ Responses returned by Verbit's Streaming Speech Recognition services have the fo
 ### Fields description
 - `"response"` - The root element in the response JSON
   - `"id"` - A unique identifier of the response (UUID)
-  - `"type"` - The response type. Can be either "transcript" or "captions" (See explanation in [README.md](../../README.md#responses)).
-  - `"is_final"` - A boolean denoting whether the response is the final one for the utterance (See explanation in [README.md](../../README.md#responses)). For a "captions" response, this is always set to `"true"`, since captions are not incrementally updated (thus, each "captions" response is final).
+  - `"type"` - The response type. Can be either "transcript" or "captions" (See explanation in [README.md](https://github.com/verbit-ai/verbit-streaming-python-sdk/blob/main/README.md#responses)).
```
Comment (kobicalev) [inline examples/responses/schema.md:41]: Not one of the current changes: but for `alternatives/start`+`end`: let's make sure they're never `null` (but actually `float`) on the server side, in the case of an empty caption reply. Also we might mention the empty caption reply, or be explicit about the option that `alternatives` is an empty array.

Comment (kobicalev) [reply examples/responses/schema.md:41]: We actually say
"The alternatives array will always have only one item for captions." (in the Readme) let's change that to zero-or one.

Comment (kobicalev) [reply examples/responses/schema.md:41]: So, it's indeed: and empty `alternatives` + main `start/end` == `0.0`;  let's change the Readme to include that though.

Comment (EliAsor) [reply examples/responses/schema.md:41]: we will always have one alternative. The `word_list` in the `alternative` can be empty, but we will always have one, and the `start/end` times will always be float. So I think it's all good.

**Review by kobicalev — APPROVED** [review]

**Pull request merged**